Content.Client/GameObjects/Components/Atmos/GasAnalyzerMenu.cs

```diff
@@ -149,20 +149,21 @@ public void Populate(GasAnalyzerBoundUserInterfaceState state)
             {
                 _statusContainer.AddChild(new Label
                 {
-                    Text = Loc.GetString("Error: {0}", state.Error),
+                    Text = Loc.GetString("gas-analyzer-window-error-text", ("errorText", state.Error)),
                     FontColorOverride = Color.Red
                 });
                 return;
             }
 
             _statusContainer.AddChild(new Label
             {
-                Text = Loc.GetString("Pressure: {0:0.##} kPa", state.Pressure)
+                Text = Loc.GetString("gas-analyzer-window-pressure-text", ("pressure", $"{state.Pressure:0.##}"))
```

Comment:
I couldn't make the :0.## formatting work in fluent, so it had to go here. Unless I missed something?